extensions/applicationinsights-properties-js/src/Context/Session.ts

```diff
@@ -169,9 +169,8 @@ export class _SessionManager {
                             _logger.throwInternal(LoggingSeverity.WARNING,
                                 _InternalMessageId.SessionRenewalDateIsZero,
                                 "AI session renewal date is 0" + sessionReset);
-                        } else if (session.id) {
-                            // If we don't have an id then considered it to be expired
-                            // Only assign the values if everything looks good
+                        } else if (tokens[0]) {
```

Comment:
This is where the bug was introduced - as the caller of this function only calls _initializeAutomaticSession() if (!session.id) -- so this was always false and subsequently got removed from the code.